doc/index.rst

```diff
@@ -52,7 +52,7 @@ Documentation outline
 
    auto_examples/index
 
-:ref:`genindex` | :ref:`modindex` | :ref:`search`
+:ref:`genindex` | :doc:`Modules <./metric_learn>` | :ref:`search`
```

Comment:
What does this do?

Reply:
This page is already linked a bit above on the index page, so perhaps we could simply remove it?
Or, fix the problem preventing the generation of this file

Reply:
I am not knowledgeable in sphinx so I can't find a quick fix, in the meantime, I will remove it.